tools/distrib/python/grpcio_tools/setup.py

```diff
@@ -44,6 +44,12 @@
 import protoc_lib_deps
 import grpc_version
 
+_EXT_INIT_SYMBOL = None
+if sys.version_info[0] == 2:
+    _EXT_INIT_SYMBOL = "init_protoc_compiler"
+else:
+    _EXT_INIT_SYMBOL = "PyInit__protoc_compiler"
```

Comment:
Let's hope ProtoBuf doesn't change the init function.

Reply:
This is actually the name of the `grpcio-tools` C extension. We're fundamentally in control of this, so there's nothing to worry about there. The only time this would need to change is if Python introduced a new ABI, which is  unlikely.